sdk/python/kubeflow/training/utils/utils.py

```diff
@@ -68,11 +76,11 @@ def get_job_labels(name, master=False, replica_type=None, replica_index=None):
     :return: Dict: Labels
     """
     labels = {
-        constants.JOB_GROUP_LABEL: 'kubeflow.org',
+        constants.JOB_GROUP_LABEL: "kubeflow.org",
         constants.JOB_NAME_LABEL: name,
     }
     if master:
-        labels[constants.JOB_ROLE_LABEL] = 'master'
+        labels[constants.JOB_ROLE_LABEL] = "master"
```

Comment:
Should we replace `master` with constants? Or we can prepare all replace-types as constants.